compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -978,6 +983,17 @@ pub trait PrettyPrinter<'tcx>:
             write!(self, "Sized")?;
         }
 
+        if let [re] = lifetimes.as_slice()
+            && re.is_static()
+        {
+            // Don't print a single static lifetime
```

Comment:
Why not?  The `'static` bound is significant when checking that the hidden type satisfies it.

Reply:
Yeah, I don't care particularly. I can add it back.